tags/Generic/TaxonomiesTracker/Taxonomy Tracker.tpl

```diff
@@ -0,0 +1,138 @@
+    "displayName": "",
+    "id": "brand_dummy"
+  },
+  "containerContexts": [
+    "WEB"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ],
+    "displayName": "Taxonomy Name",
+    "simpleValueType": true,
+    "name": "taxonomyName",
+    "type": "TEXT"
+  },
+  {
+    "macrosInSelect": true,
+    "selectItems": [],
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      },
+      {
+        "args": [],
+        "type": "TABLE_ROW_COUNT"
+      }
```

Comment:
This validation won't really do much. Validation doesn't apply to GTM variables, since those are evaluated at runtime, whereas validation is evaluated when the tag is created.

Reply:
Removed validation